src/data/careerSuccessData.ts

```diff
@@ -0,0 +1,51 @@
+export type CareerData = {
```

Comment:
Either make type `CareerSuccessData` or change the constant to `careerData` just for consistency.